docs/site/configuration.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Configuration
+nav_order: 6
+---
+# Configuration
+
+The following configurations can be supplied as Spark properties.
+
+| Property name   | Default | Meaning  |
+| :-------------- | :------ | :------- |
+| spark.rapids.ml.uvm.enabled | false | if set to true, enables [CUDA unified virtual memory](https://developer.nvidia.com/blog/unified-memory-cuda-beginners/) (aka managed memory) during estimator.fit() operations to allow processing of larger datasets than would fit in GPU memory|
```

Comment:
does spark-rapids parse this configuration?

Reply:
I don't think so.    They have an undocumented similar one without the ml.